f8a_worker/workers/CVEchecker.py

```diff
@@ -222,12 +250,61 @@ def _clean_dep_check_tmp():
                 'status': 'success',
                 'details': results}
 
+    @staticmethod
+    def update_victims_cve_db_on_s3():
+        """Update Victims CVE DB on S3"""
+        repo_url = 'https://github.com/victims/victims-cve-db.git'
+        s3 = StoragePool.get_connected_storage('S3VulnDB')
+        with tempdir() as temp_data_dir:
+            victims_db_dir = os.path.join(temp_data_dir, 'victims-cve-db')
+            Git.clone(repo_url, victims_db_dir, depth="1")
+            s3.store_victims_db(temp_data_dir)
```

Comment:
2) I presume that you want to work inside the directory where you just cloned latest victimDB.